aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1651,6 +1651,19 @@ Tensor _cholesky_helper_cuda(const Tensor& self, bool upper) {
   return upper ? result.transpose_(-1, -2) : result;
 }
 
+Tensor _cholesky_helper_cuda(const Tensor& self, bool upper) {
+#ifdef USE_CUSOLVER
+  if (batchCount(self) == 1 || !use_magma_) {
```

Comment:
Would you add a TODO comment here to review using cuSOLVER for batches once the batched cuSOLVER issue is addressed, and add a link to the tracking issue you created?